hushline/settings.py

```diff
@@ -395,6 +408,8 @@ async def index() -> str | Response:
             user=user,
             secondary_usernames=secondary_usernames,
             all_users=all_users,  # Pass to the template for admin view
+            update_brand_primary_color_form=update_brand_primary_color_form,
```

Comment:
should we pre-populate the fields with the existing db values before this return statement?

Reply:
I'm not sure. Because the tabs where the related forms are rendered require custom & coordinated attributes, I've manually defined them instead of relying completely on jinja inserts.


https://github.com/scidsg/hushline/blob/47d49da26ac78d123b0ad310de2b76f70ce2f222/hushline/templates/settings.html#L499-L507


https://github.com/scidsg/hushline/blob/47d49da26ac78d123b0ad310de2b76f70ce2f222/hushline/templates/settings.html#L516-L525


Pulling all the values from the forms & creating the whole tag + attributes using jinja could work, but that's not the existing pattern in this file. Additionally, accessing the ``host_org`` variable here

https://github.com/scidsg/hushline/blob/47d49da26ac78d123b0ad310de2b76f70ce2f222/hushline/settings.py#L286-L287

is a complication. I started with a version where ``host_org`` is accessed here, which caused db & key errors until the ``create_all()`` line in '\_\_init\_\_.py' was added https://github.com/scidsg/hushline./pull/573#discussion_r1761586224. But the ``create_all()`` line doesn't seem to be necessary when ``host_org`` is only accessed by the jinja templates.

I'm open to the idea that there's a good reason, or easy way, to rework how this is connected, but I don't think a change will be trivial.